test/verify/check-no-retry

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python3
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2020 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import parent
+from testlib import *
+
+
+@noretry
+@skipImage("Do not test BaseOS packages", "rhel-8-3-distropkg")
+class TestNoRetryClass(unittest.TestCase):
+
+      def testShouldOnlyRunOnceClass(self):
+          self.assertTrue(True)
```

Comment:
To enforce this, the test could write a stamp file into tmp/ and ensure that it's not present yet? But then again, you couldn't manually run it more than once, so nevermind.

Reply:
Yea I couldn't find a good way either, if we use a nondestructive machine we could but then we can't rely on that machine not being reset in between because test harness failures.